test/tests/bsatn-packing.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bashtable
+
+if [ "$DESCRIBE_TEST" = 1 ] ; then
+        echo "This tests the iter_filtered method in reducers."
```

Comment:
I think iter_filtered would be a better name for this test case file, if it's how you are describing the test.